akka-actor/src/main-2.13/scala/akka/compat/Future.scala

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+
+package akka.compat
+
+import akka.compat
+import scala.concurrent.{ ExecutionContext, Future ⇒ SFuture }
+import scala.collection.immutable
+
+/**
+ * Compatibility wrapper for `scala.concurrent.Future` to be able to compile the same code
+ * against Scala 2.11, 2.12, 2.13
+ */
+private[akka] object Future {
+  def fold[T, R](futures: TraversableOnce[SFuture[T]])(zero: R)(op: (R, T) ⇒ R)(implicit executor: ExecutionContext): SFuture[R] = {
```

Comment:
I'm wondering if we should add a deprecation warning for the 2.12 version of this method so it is obvious that there will be performance implications when going to 2.13